packages/vscode-extension/package.json

```diff
@@ -616,8 +616,7 @@
       },
       {
         "command": "fx-extension.deployAadAppManifestFromCtxMenu",
-        "title": "%teamstoolkit.commands.deployAadAppManifestFromCtxMenu.title%",
-        "enablement": "fx-extension.isAadManifestEnabled"
```

Comment:
Any reason to remove this?

Reply:
this is by design, no need to check the condition, and user can trigger this command anytime.
we check the condition in fx-core, and throw error on output